recipes/freetype/config.yml

```diff
@@ -5,3 +5,5 @@ versions:
     folder: all
   "2.10.2":
     folder: all
+  "2.10.4":
+    folder: all
```

Comment:
minor change: Could you reorder versions in `config.yml` like you did for `conandata.yml` in https://github.com/conan-io/conan-center-index/commit/d014f70febdf2e32064a8a59ea7427dcfbfea443?